src/client.ts

```diff
@@ -55,8 +55,37 @@ export default class UnleashClient extends EventEmitter {
     }
   }
 
+  isParentDependencySatisfied(feature: FeatureInterface | undefined, context: Context) {
+    if (!feature?.name) {
```

Comment:
Can feature not have name?

Reply:
this is what the existing type was telling me so I'm respecting it here

Reply:
Feature's should have a name, but at least in this SDK we allow people to define their own bootstrapper so it's possible people put goop in there. This one is probably mostly legacy but respecting the existing types seems very sane to me